sweep/sweeper.go

```diff
@@ -305,6 +305,10 @@ type PendingInputResponse struct {
 
 	// DeadlineHeight records the deadline height of this input.
 	DeadlineHeight uint32
+
+	// MaturityHeight is the block height that this input's locktime will
+	// be expired at. For inputs with no locktime this value is zero.
+	MaturityHeight uint32
 }
 
```

Comment:
which change allowed us to now bump the fee of the immature inputs, I cannot see a change here made which would change this behaviour?

Reply:
If an input is immature we cannot bump it, since it cannot be spent yet. The change here is really simple, we just return all inputs in `PendingSweeps`, so the user can plan ahead. For instance, the user may want to group the `to_local` and outgoing HTLCs together - previously they couldn't until the timelock expires, now they can update the `deadline` for these immature inputs, as manifested in the update itest, where we now always have the immature inputs returned.

Reply:
ahh gottya, we also call PendingSweeps when using bumpFee rpc, ok makes sense, thank you for clarification.